src/java.base/share/classes/java/time/OffsetTime.java

```diff
@@ -1281,7 +1281,11 @@ public long toEpochSecond(LocalDate date) {
      * use {@link ChronoField#NANO_OF_DAY} as a comparator.
      *
      * @param other  the other time to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value, that is the comparison of the UTC equivalent {@code other} instant,
+     *          if they are not equal, and if the UTC equivalent {@code other} instant is equal,
+     *          the comparison of this local date-time with {@code other} local date-time
```

Comment:
This text is incorrect. It compares the local time, not the local date-time, when  the instants are equal